packages/bits/demo/src/.eslintrc.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
is there a reason why the demo app needs two .eslintrc files? I wonder if they could be combined into a single file.

if two files are indeed needed, maybe some of the properties in this file could be omitted if they have the same value as the one being extended.

Reply:
.eslintrc.json files are generated for each angular project respectively. Since both "demo" and "demo-e2e" projects use the same "docs" root directory, but different source directories, they're generated in different places. They're only different with a specific tsconfig file, the rest of the properties are mandatory.

I tried to play around with the project source folders and config files and could only make it work the default way schematics generate those files. I need more time to investigate this, if it's a priority I'm fine with having a task to investigate.

Reply:
Okay, the current setup is alright. I wouldn't consider figuring out a one-file solution to be a priority.